### ISSUE TITLE
Mortar Beta 0.2.0-beta0007 Cannot find Templates

### ISSUE THREAD

**Issue by tsmarvin** [JustinGrote/Mortar#1]
Howdy Justin, 

I decided I'd try to give Mortar/Press a shot on a new project I'm working on but ran into an issue with the 0.2.0-beta0007 version that I installed from the gallery. 

![image](https://user-images.githubusercontent.com/57049894/123345549-29593200-d50b-11eb-9064-e2b47f1c44a5.png)

Line 58 of Mortar.psm1: 
```ps1
$SCRIPT:templatePath = if (Test-Path "$PSScriptRoot/../Templates") {Resolve-Path "$PSScriptRoot/../Templates"} else {Resolve-Path 'Templates'}
```

The logic as written works if you download the repo from github - but not when it's installed from the gallery. 

I manually changed line 58 in Mortar.psm1 to: 
```ps1
$SCRIPT:templatePath = if (Test-Path "$PSScriptRoot/Templates") {Resolve-Path "$PSScriptRoot/Templates"} else {Resolve-Path 'Templates'}
```
and it solved the problem for me. 

I didn't open a pull request for this one since it works with the repo configuration - and you've got the todo note right above it - but figured I'd let you know.

**Comment by JustinGrote** [comment]
Yeah my plan is to make this pluggable and allow Github Repos/Modules to supply their own templates. It may be a bit as my main focus is on the Pester test adapter for vscode right now.

I want to emphasize both Mortar/Press are in barely alpha state right now.
